components/rhel/misc/hyperv/rhel-k8s-singlenode-hyperv/README.rst

```diff
@@ -0,0 +1,36 @@
+Vagrantfile: Single Node Kubernetes
+===================================
+
+This Vagrantfile is the suggested configuration for using the CDK with Kubernetes.  
+
+This file expects an external type virtual switch for HyperV. The Vagrantbox IP should be used to expose the ``docker`` daemon and ``Kubernetes`` to the host.  
+
+This Vagrantfile also sets up a single node Kubernetes instance where the master and node are on the same machine.
+
+This Vagrantfile is useful for anyone using host-based tools, such as the `Eclipse docker tooling <https://wiki.eclipse.org/Linux_Tools_Project/Docker_Tooling>`_ or the kubectl, oc and docker CLIs, with the CDK.
+
+QuickStart
+----------
+
+1. Get latest CDK box and add it to vagrant.
+
+2. Create a directory for the Vagrant Box
+
+  ``$ mkdir directory && cd directory``
+
+3. Use this Vagrantfile
+
```

Comment:
@Preeticp It would be part of the CDK.zip. So I think we do not need to give explicit instruction about it.